src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -141,11 +141,12 @@ private void testActions() throws Exception {
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_PIC_TOO_LARGE);
         verifyPictureIsPresent(prevPictureKey);
         
-        ______TS("success case, update picture");
+        ______TS("success case, update picture (to long)");
```

Comment:
hahaha sorry sir meant to say "updated picture to a long one". will update it :-)